src/client/Python/msrest/readme.rst

```diff
@@ -15,6 +15,13 @@ To install:
 Release History
 ---------------
 
+2016-09-01 Version 0.4.3
+++++++++++++++++++++++++
+
+**Bugfixes**
+
+- Better exception message (https://github.com/Azure/autorest/pull/1300)
```

Comment:
also the fix is in msrestazure, not the generic msrest, so we probably should mention in msrestazure's readme